x-pack/plugin/transform/src/test/java/org/elasticsearch/xpack/transform/transforms/TransformTaskTests.java

```diff
@@ -616,6 +616,6 @@ private void testTriggered(TransformTaskState taskState, IndexerState indexerSta
     }
 
     private static TransformTaskParams createTransformTaskParams(String transformId) {
-        return new TransformTaskParams(transformId, TransformConfigVersion.CURRENT, TimeValue.timeValueSeconds(10), false);
+        return new TransformTaskParams(transformId, TransformConfigVersion.CURRENT, TimeValue.timeValueSeconds(10), false, false);
```

Comment:
It is probably ok to pass `unattended`: `false` here.
Just please keep in mind that the config for which we are creating these task params is created randomly so it is possible that it has set `unattended` to `true`.